packages/keyda-demo/src/lib/index.js

```diff
@@ -0,0 +1,2 @@
+export { default as Input } from './Input';
+export { default as Form } from './Form';
```

Comment:
The directory `lib` comes from `keyda-react` temporarily for developing library functions in a browser environment. So it should be deleted after all of the work is done.